src/common-utils/install.sh

```diff
@@ -378,6 +379,12 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
         ZSH_ALREADY_INSTALLED="true"
     fi
 
+    # Change shell of determined user
+    if [[ "${CHANGE_DEFAULT_SHELL_ZSH}" == "true" ]]; then
+      chsh --shell /bin/zsh ${USERNAME}
```

Comment:
Could you write a test scenario and help validate this change? Thanks! 

Ref doc - https://github.com/devcontainers/cli/blob/main/docs/features/test.md
Sample - [dotnet tests](https://github.com/devcontainers/features/tree/1c60fd71daa84c5fb81b4daf499974dbffb9e029/test/dotnet)

Reply:
Committed, please let me know if it needs amending